src/actions.cpp

```diff
@@ -555,6 +564,68 @@ bool evm_contract::gc(uint32_t max) {
     return state.gc(max);
 }
 
+void evm_contract::call_(intx::uint256 s, const bytes& to, intx::uint256 value, const bytes& data, uint64_t gas_limit, uint64_t nonce) {
+    const auto& current_config = _config.get();
+
+    Transaction txn;
+    txn.type = TransactionType::kLegacy;
+    txn.nonce = nonce;
+    txn.max_priority_fee_per_gas = current_config.gas_price;
+    txn.max_fee_per_gas = current_config.gas_price;
+    txn.gas_limit = gas_limit;
+    txn.value = value;
+    txn.data = Bytes{(const uint8_t*)data.data(), data.size()};
+    txn.r = 0u;  // r == 0 is pseudo signature that resolves to reserved address range
+    txn.s = s;
+
+    // Allow empty to so that it can support contract creation calls.
+    if (!to.empty()) {
+        ByteView bv_to{(const uint8_t*)to.data(), to.size()};
+        txn.to = to_evmc_address(bv_to);
+    }
+
+    Bytes rlp;
+    rlp::encode(rlp, txn);
+    pushtx_action pushtx_act(get_self(), {{get_self(), "active"_n}});
+    pushtx_act.send(get_self(), rlp);
+}
+
+void evm_contract::call(eosio::name from, const bytes& to, uint128_t value, const bytes& data, uint64_t gas_limit) {
```

Comment:
I think we should change the interface to use `bytes` instead of `uint128_t` for the value parameter.
Even if its bigger than needed, i think its safer and less error prone when working with it.
We can stick to `uint64_t` for gas_limit.

Reply:
fixed